cli.js

```diff
@@ -331,6 +333,26 @@ function exportIcons () {
   })
 }
 
+function exportImages () {
```

Comment:
creating separate function here, even though its basically the same thing as exportIcons(). 

I toyed around with making a config object to pass to exportIcons(), but I decided not to prematurely optimize.